include/openmc/surface.h

```diff
@@ -86,6 +86,8 @@ class Surface {
   int id_;                                          //!< Unique ID
   std::string name_;                                //!< User-defined name
   std::shared_ptr<BoundaryCondition> bc_ {nullptr}; //!< Boundary condition
+  std::double_t bc_alb_;                            //!< Boundary albedo
```

Comment:
Don't store this on the surface; you should instead store it on the boundary condition object.

Reply:
I've updated the pull request. Moved it out of the `Surface` class